index.html

```diff
@@ -58,9 +65,9 @@ <h3 class="contact">LET'S CONNECT </h3>
 
     <section class="Projects">
         <div class="card">
-
+            <img class="snapshot" src="images/SS0.png" alt="snapshot">
             <div class="right">
-                <img class="snapshot" src="images/S0.png" alt="snapshot">
+
                 <h2>Tonic</h2>
                 <ul class="detail">
                     <li class="main">CANOPY</li>
```

Comment:
- Great job so far @Vishal30M There are just a few changes we need to make and your project will be complete :+1: 
- [ ] Each of your cards is missing some styles as seen in the Figma design. As a result, they don't look exactly like the Figma design. I have pointed them out in the image below so kindly fix these :wink: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/230685187-0c85cff8-0527-41dd-bc18-e372ddf73dfc.png)

- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/230685272-34aa3e78-b0a7-4679-96dd-688ccd376609.png)